app/app.py

```
@@ -726,110 +816,134 @@ def export_pdf():
+        elements.append(Spacer(1, 0.2 * inch))
 
         # Report info
-        generated_time = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
-        elements.append(
-            Paragraph(f"Generated: {generated_time}", styles['Normal'])
-        )
-        elements.append(
-            Paragraph(f"Session ID: {session_id[:8]}...", styles['Normal'])
-        )
-        elements.append(Spacer(1, 0.3*inch))
+        generated_time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)